openc3/lib/openc3/utilities/authentication.rb

```diff
@@ -75,14 +75,14 @@ def initialize(url)
     end
 
     # Load the token from the environment
-    def token(include_bearer: true)
+    def token(include_bearer: true, openid_scope: 'openid')
```

Comment:
What is this scope? I assume it has something to do with keycloak? Why does it now have to be configurable?

Reply:
This is the "scope" of the token.   I need to modify it to include "offline_access" to get the offline access token.